docs/modules/operation/pages/deep-dive/service-assurance/polling-packages.adoc

```diff
@@ -151,18 +151,18 @@ The following shows which configuration is applied for a specific service:
   <downtime interval="600000" begin="43200000" end="432000000" />
 </package>
 ----
-<1> Polling interval in the packages are 300 seconds and 30 seconds.
+<1> Polling interval in the packages is 300 seconds and 30 seconds.
 <2> Different polling interval for the ICMP service.
 <3> Different retry settings for the ICMP service.
 <4> Different timeout settings for the ICMP service.
 
 The last polling package on the service will be applied.
 You can use this to define a less specific catch-all filter for a default configuration.
 Use a more specific polling package to overwrite the default setting.
-In the above example, all IP interfaces in 192.168.1/24 or 2600:/64 will be monitored with ICMP with different polling, retry, and timeout settings.
+In the above example, all IP interfaces in `192.168.1/24` or `2600:/64` will be monitored with ICMP with different polling, retry, and timeout settings.
 
 The web UI displays which polling packages are applied to the IP interface and service.
-The IP Interface and Service pages show which polling package and service configuration are applied for this specific service.
+The IP interface and service pages show which polling package and service configuration are applied for this specific service.
```

Comment:
The second line here seems redundant. I'd suggest removing it or combining the two.